### ISSUE TITLE
Scroll

### ISSUE THREAD

**Issue by VldMrgnn** [juliankrispel/react-text-selection-popover#38]
I am using the popover over a Devextreme dxList component and I didn't found a solution for scrollRef to work.
If I pass a scrollView ref or a list ref the program freeze with no error.
If I wrap it in div, the scroll event is not caught in popover.
I have tried various wrapping and so on. Nothing seems to trigger the scroll event in that scenario.

**Comment by VldMrgnn** [comment]
I managed somehow to solve  the issue, but I am unhappy with what I came up with. It works for now, but I am not sure that I should go to production with it:
```
  /*
       * Before you ask, onSelectionChange only works on the document,
       * otherwise I would just use selectionRef instead and we wouldn't need
       * three of those event listeners
       */
      return [
      /// how to cleanup this event listener? Should it be removed???    
      document.addEventListener('scroll',this.handleStuff ,true),  // <-------------
      React__default.createElement(EventListener, {
        key: "update-position",
        target: document,
        onSelectionChange: this.updatePosition
      }),
   ...................................................
    , selectionPosition == null || !isOpen || contentRect.bounds.width == 0 ? null : React__default.createElement(
        "div",
        { key: "popup", className: className, style: style, ref: measureRef },
        children
      )]
```
It was impossible for me to listen to scroll event otherwise..
What am I missing here?

**Comment by juliankrispel** [comment]
@VldMrgnn I've just released 2.0 and this shouldn't really be an issue anymore. Lmk if you still need help
